spec/anchor.spec.js

```diff
@@ -68,7 +69,11 @@ describe('Anchor Button TestCase', function () {
             });
             expect(editor.createLink).toHaveBeenCalled();
             // A trailing <br> may be added when insertHTML is used to add the link internally.
-            expect(this.el.innerHTML.indexOf('<a href="http://test.com">lorem ipsum</a>')).toBe(0);
+            if (!isFirefox()) {
+                expect(this.el.innerHTML.indexOf('<a href="http://test.com" target="_self">lorem ipsum</a>')).toBe(0);
+            } else {
+                expect(this.el.innerHTML.indexOf('<a target="_self" href="http://test.com">lorem ipsum</a>')).toBe(0);
+            }
```

Comment:
Yes, firefox and chrome places target="_self" (or blank) at different positions. and thus need to add a check like this.